vtex-apps/binding-selector
# Hide labels

#### What problem is this solving?


This PR allows user to hide a label in the dropdown menu in the store through the admin panel.

#### How to test it?


In this [workspace](https://hidelabel--muji.myvtex.com/admin/binding-selector), open the `customize store name` menu and set a store to be hidden clicking in the `hide` checkbox. Visit the store for that binding and check the binding selector dropdown menu. The ones selected to be hidden should not be in the list.

[Workspace](Link goes here!)

#### Screenshots or example usage:


![recording (1)](https://user-images.githubusercontent.com/38737958/119470041-3803cc00-bd48-11eb-8157-a9f2ee935cb6.gif)
